protos/protos/public/modeldb/DatasetVersionService.proto

```diff
@@ -77,6 +77,8 @@ message DatasetPartInfo {
     uint64 size = 2;
     string checksum = 3;
     uint64 last_modified_at_source =4;
+    string version_id = 5;
+    VersionIDTypeEnum.VersionIDType version_type = 6;
```

Comment:
I wouldn't bother adding to the previous system since that might change significantly. We can add later if it turns out it won't.

Reply:
may be we have it in protos and deprioritize the implementation?

Reply:
I'd prefer not have something in protos until we actually plan to implement it. Same reason that we commented out some fields in the versioning protos.